subgraphs/compound-forks/src/mapping.ts

```diff
@@ -589,6 +602,22 @@ export function _handleBorrow(
   account.borrowCount += 1;
   account.save();
 
+  //
+  // track unique borrowers
+  //
+  let borrowerActorID = "borrower".concat("-").concat(account.id);
+  let borrowerActor = _ActorAccount.load(borrowerActorID);
+  if (!borrowerActor) {
+    borrowerActor = new _ActorAccount(borrowerActorID);
+    borrowerActor.save();
+
+    protocol.cumulativeUniqueBorrowers += 1;
+    protocol.save();
+  }
+
+  //
+  // update position
+  //
```

Comment:
How does moving the counter up here resolve the issue? I don't see any early returns in `addPosition` that could cause the counter not to be increased. The only scenario would be if it was a borrow/deposit on an already existing position, but then that account would already be accounted for. Can someone deposit into someone else's position?

Borrows and deposits share positions or are independent?

Reply:
I doesn't change the logic I guess, but it is more in line with the "precedent" since liquidation user counts are kept in the handler.
I don't believe compound allows for third parties to act on an accounts behalf.
deposits/borrows are independent, but that shouldn't matter.

Reply:
I was asking about someone else depositing on your position because that would add an extra user to the total unique users. Which by moving the code to where you moved it we would be accounting for. So I thought that was maybe what you were attempting to fix with this change :)

Also asked about positions being independent because if they were shared we would have been miscounting borrowers. And same, this change would resolve it.

Understood then that this change in particular is only for consistency purposes with the rest of the code 👍